build/int.cloudbuild.yaml

```
@@ -466,6 +466,21 @@ steps:
     - verify simple-autopilot-private-non-default-sa
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'cft test run TestSimpleAutopilotPrivateNonDefaultSA --stage teardown --verbose']
+- id: apply simple-autopilot-private-cmek
+  waitFor:
+    - create-all
```

@romanini-ciandt - for this repo

```suggestion
    - init-all
```